src/Ledger/PDF.lagda

```diff
@@ -7,7 +7,7 @@
 \NewEnviron{Conway}{%
 }
 
-\include{preamble}
+\input{preamble}
```

Comment:
IIRC `include` is only for 'subdocuments' and includes some formatting directions. So for anything that's not being shown you should use `input` because it just puts it there verbatim. Anyway, it just fixes a warning.